rl4co/envs/routing/cvrptw/env.py

```diff
@@ -114,7 +110,11 @@ def get_action_mask(td: TensorDict) -> torch.Tensor:
         current_loc = gather_by_index(td["locs"], td["current_node"]).reshape(
             [batch_size, 2]
         )
-        dist = get_distance(current_loc, td["locs"].transpose(0, 1)).transpose(0, 1)
+        dist = (
+            get_distance(current_loc, td["locs"].transpose(0, 1))
```

Comment:
You can achieve the same as above with the `v2` which is a bit more readable and faster:

```python
from tensordict import TensorDict
import torch

from rl4co.utils.ops import gather_by_index, get_distance

td = TensorDict(
    {
        "locs": torch.rand(32, 10, 2),
        "current_node": torch.randint(0, 10, (32,1))
    },
    batch_size = 32
)

def get_distv1():
    batch_size = td["locs"].shape[0]
    current_loc = gather_by_index(td["locs"], td["current_node"]).reshape(
        [batch_size, 2]
    )
    dist = (
        get_distance(current_loc, td["locs"].transpose(0, 1))
        .transpose(0, 1)
        .contiguous()
    )
    return dist

def get_distv2():
    cur_loc = gather_by_index(td["locs"], td["current_node"])
    d_ij = get_distance(cur_loc[..., None, :], td["locs"])
    return d_ij

torch.allclose(get_distv1(), get_distv2())
```